mathgenerator/algebra.py

```diff
@@ -168,7 +180,7 @@ def distance_two_points(max_val_xy=20, min_val_xy=-20):
 
     distanceSq = (point1X - point2X)**2 + (point1Y - point2Y)**2
 
-    solution = rf"$\sqrt{{{distanceSq}}}$"
+    solution = str(round(distanceSq**0.5, 2))
```

Comment:
random thought - would be super cool if we create the challenge based on a real world version of this problem. e.g. "I have a garden hose which is 5m and need to water the flowers at the other side of my garden - will it reach?"

Reply:
I like that, do you envision that this would be controlled by the agent?